bio/kraken2/classify/wrapper.py

```
@@ -0,0 +1,99 @@
+paired = fq2 is not None
+if paired:
+    fq2 = [fq2] if isinstance(fq2, str) else list(fq2)
+    assert len(fq1) == len(
+        fq2
+    ), "Input -> equal number of files required for fq1 and fq2"
+
+input_str_db = ",".join(db)
+input_str_fq1 = " ".join(fq1)
+input_str_fq2 = " ".join(fq2) if paired else ""
+
+db_files = [snakemake.input["hash"], snakemake.input["opts"], snakemake.input["taxo"]]
```

Ruff (0.15.7)

[error] Undefined name `snakemake`

(F821)